internal/internal_task_pollers.go

```diff
@@ -465,6 +465,9 @@ func (lath *localActivityTaskHandler) executeLocalActivityTask(task *localActivi
 	}
 
 	ctx := context.WithValue(rootCtx, activityEnvContextKey, &activityEnvironment{
+		workflowType:      &task.params.WorkflowInfo.WorkflowType,
```

Comment:
just to be defensive, consider making a copy in a local var and then pass the addr of that local var here